packages/react-core/src/components/Tabs/OverflowTab.tsx

```diff
@@ -21,6 +21,8 @@ export interface OverflowTabProps extends React.HTMLProps<HTMLLIElement> {
   toggleAriaLabel?: string;
   /** z-index of the overflow tab */
   zIndex?: number;
+  /** Flag indicating if scroll on focus of the first menu item should occur. */
+  preventScrollOnItemFocus?: boolean;
```

Comment:
nit. Same comment here about the name of this prop. Can we prefix it with `should`?